docs/source/release/v5.1.0/index.rst

```diff
@@ -64,7 +64,7 @@ Changes
 
   - :doc:`MantidPlot <mantidplot>`
 
-  - :doc:`Workbench <mantidworkbench>`
+  - :doc:`MantidWorkbench <workbench>`
```

Comment:
Thanks for updating this. Could we revert everywhere to say `mantidworkbench` like it used to? That way just have the single name everywhere that is consistent.